WEB_SERVICE/WEB/_HIGH/4XX_BYPASS/description.md

```diff
@@ -0,0 +1,32 @@
+
+# 4XX-Bypass
+
+The 4xx-bypass exploits refer to HTTP request fuzzing techniques that allow attackers to bypass server-side restrictions, accessing unauthorized resources or performing malicious actions on a web application. This can lead to unauthorized data access, privilege escalation, and other security risks.
```

Comment:
Nah nah nah. There is no such thing as a 4xx bypass. Call it Insecure Authorization Restriction and change all the entries.